### DISCUSSION TITLE
Routenberechnung

### DISCUSSION THREAD

**Discussion by overheaddesign** [Tastaturberuf/anyStores#104] [Feature Requests / Funktionswünsche]
Zusätzlich zu der Google Map einfach noch ein Feld wo man die Startadresse eingeben kann (Ziel Adresse ist der jeweilige Händler) und das man dann die Route berechnen lassen kann per externen Googlemap.

**Comment by Tastaturberuf** [comment]
Hab was interessantes gefunden, würde ich gerne so implementieren:

![screenclip](https://cloud.githubusercontent.com/assets/1027521/6769390/88ac300e-d099-11e4-8606-37d0276e013b.png)

http://www.winlocal.de/ueber-uns/buro-berlin/

**Comment by tim-bec** [comment]
Lässt sich das nicht einfach mit der Verwendung der neuen google-maps api realisieren? Das ist ja ein feature von gmaps

**Comment by Tastaturberuf** [comment]
Ich hab’s mir im Detail noch nicht angesehen.

**Comment by Tastaturberuf** [comment]
Denke das kommt nur wenn man das per iframe einbindet.

**Comment by gassi101** [comment]
gibts hier schon einen Fortschritt? Fände das Feature auch super... Würde es nicht auch so gehen - es wie bei dem Standard Googlemaps Modul zu realisieren? Also "einfach" im Infofenster eine Input-Zeile für die Adresse und dann ein link  "Route berechnen" ?

**Comment by stefansl** [comment]
@gassi101 Das ist kein großes Ding. Leg einfach eine neue Version vom Template mod_anystores_map.html5 an und ändere dieses wie hier im Patch:
https://github.com/Tastaturberuf/anyStores/compare/master...stefansl:patch-1

Edit: Genauso wird es auch bei dlh_googlemaps umgesetzt.

**Comment by stefansl** [comment]
Hier auch ein Beispiel: http://demo.clickpress.de/index.php/tests.html

**Comment by gassi101** [comment]
Hey Stefansl,

TIPP TOPP - Muchas Gracias!!!!

**Comment by stefansl** [comment]
Gerne :)
